examples/run_examples.py

```diff
@@ -0,0 +1,52 @@
+from crapssim.table import Table, TableUpdate
+from crapssim.strategy.examples import (
+    QuickProps,
+    BuySampler,
+    LaySampler,
+    PutWithOdds,
+    HornExample,
+    WorldExample,
+)
+
+# Fixed roll sequence to exercise typical paths:
+# - Set point ON at 6, hit 6/8, toss a 7, then a horn number, then 4/10.
+ROLLS = [(3, 3), (4, 4), (4, 3), (1, 1), (2, 2), (6, 4)]
+
+
+def run_example(name, strategy_factory):
+    print(f"\n=== {name} ===")
+    table = Table()
+    player = table.add_player()
+    player.strategy = strategy_factory()
+
+    for die_one, die_two in ROLLS:
+        TableUpdate.roll(table, fixed_outcome=(die_one, die_two), verbose=False)
```

Comment:
I think this is actually not running the strategies and doing the roll all the way.

Can replace both lines with `table.fixed_run(dice_outcomes=ROLLS, verbose=False)`